webui/ocw/lib/EC2.py

```diff
@@ -1,23 +1,30 @@
+from .provider import Provider
+from .vault import EC2Credential
+from distutils.version import LooseVersion
+from dateutil.parser import parse
+from datetime import datetime
+from datetime import timedelta
+from datetime import timezone
 import boto3
+import re
 import time
-from .vault import EC2Credential
 import logging
 
+logger = logging.getLogger(__name__)
```

Comment:
why ?

Reply:
I thought it is to complicated to make the logger "object" specific. 
From my point of view this is the more common way and currently we don't use any specific logger config at all.